framework/src/media/streaming/audio_decoder.cpp

```diff
@@ -621,6 +649,147 @@ bool aac_check_type(rbstream_p rbsp)
 	return result;
 }
 
+// Initialize the wav reader.
```

Comment:
How about seperate this to another file like a others?(wav.cpp in codec folder)

Reply:
good idea, aduio_decoder.cpp is huge now, I will seperate code.. thanks

Reply:
I seperated wave related code to codecs/wav_decoder_api.cpp.
Plan to seperate mp3, aac code later.